cst438_gradebook/src/main/java/com/cst438/services/RegistrationServiceMQ.java

```diff
@@ -57,7 +58,13 @@ public void sendFinalGrades(int course_id, FinalGradeDTO[] grades) {
 		 
 		System.out.println("Start sendFinalGrades "+course_id);
 
-		//TODO convert grades to JSON string and send to registration service
+		System.out.println("Start sendFinalGrades " + course_id);
+
+	    // Convert grades to a JSON string and send it to the registration service
```

Comment:
This is all correct as far as I know but you can do it on one line too by writing `rabbitTemplate.convertAndSend(registrationQueue.getName(), asJsonString(grades));` . 
Its not necessary, but I think it makes the code look nicer.